packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -87,7 +111,7 @@ export const getPublicEvent = async (
   const usernameList = getUsernameList(username);
   const orgQuery = org
     ? {
-        slug: org ?? null,
+        OR: [{ slug: org }, { metadata: { path: ["requestedSlug"], equals: org } }],
```

Comment:
Needed to make sure we also selected an unpublished organization to not end up in a 404 page